eventhandlers.go

```diff
@@ -15,6 +15,9 @@ import (
 	keptn "github.com/keptn/go-utils/pkg/lib"
 )
 
+// SendTestsFinishedEvent defines whether to send a test finished event after executing chaos tests
+var SendTestsFinishedEvent = os.Getenv("SEND_TEST_FINISHED_EVENT")
```

Comment:
Can we have a default defined within the code, just in case the ENV is not included in the deployment. We can use the `ok.LookupEnv` for that

```golang
var (
    DefaultChaosTestsFinishedEventGeneration = "false"
)
```
```golang
var SendTestsFinishedEvent = getEnv("SEND_TEST_FINISHED_EVENT", DefaultChaosTestsFinishedEventGeneration)
```

```golang
func getEnv(key, fallback string) string {
    value, exists := os.LookupEnv(key)
    if !exists {
        value = fallback
    }
    return value
}
```